### ISSUE TITLE
Add flag to run on CPU/GPU explicitly

### ISSUE THREAD

**Issue by kamac** [hanxiao/bert-as-service#108]
Currently you're doing this:

```
try:
    import GPUtil
    available_gpus = GPUtil.getAvailable(limit=self.num_worker)
    if len(available_gpus) < self.num_worker:
        self.logger.warn('only %d GPU(s) is available, but ask for %d' % (len(available_gpus), self.num_worker))
except FileNotFoundError:
    self.logger.warn('nvidia-smi is missing, often means no gpu found on this machine. '
                     'will fall back to cpu!')
run_on_gpu = False
```

https://github.com/hanxiao/bert-as-service/blob/3b598e1a49121639011fe564a635ddfd4d23f521/server/bert_serving/server/__init__.py#L103

There are two problems with it:

1. If I have GPUs but want to run on CPU, this won't work for me (number of workers will be limited by the number of GPUs I have)
2. If I have GPUs but want to run on CPU and my GPUs are busy, this will limit number of workers to 0, and nothing will be run.

Solution:
Running on GPU/CPU should be set by a flag so the above code won't fire if I want to run on CPU.

**Comment by hanxiao** [comment]
well said, will fix

**Comment by hanxiao** [comment]
fyi, this issue is fixed by adding a `-cpu` flag on the `bert-serving-start` CLI and is available since `1.5.0`. please do `pip install -U bert-serving-server bert-serving-client`. Thanks again!
